ui/components/MesherySettingsPerformanceComponent.js

```diff
@@ -2,6 +2,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import Button from '@material-ui/core/Button';
+import SaveIcon from '@material-ui/icons/Save';
```

Comment:
Hey @sayantan1413, i think we are supposed to use the non-filled save icon. The save icon you have used is a filled one as far as i know, try importing 'SaveOutlinedIcon' instead

Reply:
<img width="157" alt="Screenshot 2021-07-30 at 1 24 41 AM" src="https://user-images.githubusercontent.com/64970095/127557349-5563f6a9-f743-4318-8adc-b66fecd52e0a.png">

You mean this one right?

Reply:
yes

Reply:
Thank you, gents. A little more space between the icon and word, would be great.

Reply:
Yes @leecalcote sure I have added a space between the icons and the text.